app/src/Classes/Model.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace src\Classes;
+
+use Exception;
+
+abstract class Model {
+    private string $data;
+
+    private string $id;
+
+    abstract function getAll();
+
+    abstract function getItemByID(string $id);
+
+    public function errorHandler(string $message, int $code): never
```

Comment:
not sure that it is the right place to handle it

Reply:
changed